paymentsheet/src/main/java/com/stripe/android/paymentsheet/FlowControllerCompose.kt

```diff
@@ -108,7 +108,7 @@ private fun internalRememberPaymentSheetFlowController(
         "PaymentSheet.FlowController must be created in the context of an Activity"
     }
 
-    return remember {
+    return remember(paymentOptionCallback, paymentResultCallback) {
```

Comment:
From https://github.com/stripe/stripe-android/pull/8466, @jaynewstrom-stripe you suggested including createIntentCallback, externalPaymentMethodConfirmHandler in here as well. Those are updated in their own `LaunchedEffect`s which already should re-run when they change. I don't think we actually need to recreate PaymentSheet + FlowController as well. Lmk if I'm missing something though!

Reply:
Seems reasonable.